tests/unit/footer/Footer.spec.js

```diff
@@ -1,8 +1,8 @@
 import { shallowMount } from '@vue/test-utils'
-import HelloWorld from '@/components/HelloWorld.vue'
+import Footer from '@/components/footer/Footer.vue'
```

Comment:
Please put the components tests under a components folder
tests/unit/components/footer/Footer.spec.js